Assets/Scripts/NPC's/FriendlyNPC/OxygenTankPickup.cs

```diff
@@ -20,10 +21,11 @@ private void OnTriggerEnter(Collider other)
             Speech.text = "";
             Objectives.text = "Return the oxygen tank to the diver";
             BoxCollider boxCollider = Diver.GetComponent<BoxCollider>();
+            OxygenTankIcon.gameObject.SetActive(true);
             Destroy(boxCollider);
 
             // Unlock the achievement
-            CollectableManager.Instance.UnlockCollectable("oxygen_pickup");
+            //CollectableManager.Instance.UnlockCollectable("oxygen_pickup");
```

Comment:
So is there no longer a collectable/achievement for the oxygen tank?

Reply:
Line of code causes error -> needs fixing